docs/Halogen-Events.md

```diff
@@ -330,10 +338,21 @@ import Control.Functor (($>))
 H.a (E.onclick \_ -> E.preventDefault $> ClickHandler) (H.text "Click here")
 ```
 
+#### `EventHandler`
+
+``` purescript
+type EventHandler = EventHandlerT Identity
+```
+
+`EventHandler` is a synonym for `EventHandlerT` applied to the `Identity` monad.
+
+That is, `EventHandler` only adds the `preventDefault`, `stopPropagation`,
```

Comment:
I don't understand this description. Those functions produce `EventHandlerT`, not `EventHandler`.